tests/Fixer/ControlStructure/NoUselessElseFixerTest.php

```diff
@@ -22,6 +22,21 @@
  */
 final class NoUselessElseFixerTest extends AbstractFixerTestCase
 {
+    public function testLocalheinz()
+    {
+        $input =
+            '<?php
+                return $tmp(function () use $a {
+                    if ($a) {
+                        $a++;
+                    } else {
+                        $a--;
+                    }
+                };
```

Comment:
thanks! I'm re-doing this whole fix ATM, what I need is a limit on the `getTokenOfKindSibling` method. Hope to fix this evening!